explainaboard/analysis/analyses.py

```diff
@@ -391,11 +431,16 @@ def perform(
             if x not in cases[0].features:
                 raise RuntimeError(f"combo analysis: feature {x} not found.")
 
-        combo_map: dict[tuple[str, ...], int] = {}
+        combo_map: defaultdict[tuple[str, ...], list[int]] = defaultdict(list)
         for case in cases:
             feat_vals = tuple([case.features[x] for x in self.features])
-            combo_map[feat_vals] = combo_map.get(feat_vals, 0) + 1
-        combo_list = list(combo_map.items())
+            combo_map[feat_vals].append(case.sample_id)
+
+        combo_list = [
+            ComboOccurence(k, self._subsample_analysis_cases(self.sample_limit, v))
```

Comment:
Because `ComboOccurence` doesn't store the number of elements in the original list, limiting the number of elements in the list to `self.sample_limit` will change the result of the confusion matrix, right? I think `ComboOccurrence` probably needs another member `total_count`, which is set to `len(v)` before subsampling.

Reply:
Oh you are right! I didn't realize this. I somehow assumed `len(sample_ids)` and `total_count` would always be the same thing. Will fix this right away. Thanks!